scipy/stats/morestats.py

```diff
@@ -3478,3 +3478,76 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         R = np.minimum(1, hypot(sin_mean, cos_mean))
 
     return ((high - low)/2.0/pi) * sqrt(-2*log(R))
+
+
+def quartile_coeff_dispersion(a, q=(0.25, 0.75), axis=0,
+                              interpolation='lower'):
+    """
+    Compute the quartile coefficient of dispersion for a given sample array.
+
+    Given two quartiles Q_low, Q_high from array samples such that
+    Q_low < Q_high, the quartile coefficient of dispersion is calculated by
+    (Q_high - Q_low)/(Q_high + Q_low).
+    by default it is computed using 3rd and 1st quartiles.
+
+    Parameters
+    ----------
+    a : array_like
+        Input array.
+    q : array_like of float, optional
+        Sequence of 2 quantiles, such that q[0] < q[1].
+        The default is to compute with Q_high = 0.75, Q_low = 0.25.
```

Comment:
```suggestion
        Sequence of 2 quantiles, (Q_low, Q_high), such that Q_low < Q_high.
        The default is to compute with Q_low = 0.25, Q_high = 0.75
```
First line suggestion is because relationship between `q` and (Q_low, Q_high) was not explicitly established here. I think this is just slightly more clear. 
Second is because Q_low appears first everywhere else (except as necessary in the formula). 100% personal nitpick; so you're welcome to disagree if you prefer the other way.